libs/filer/local_client.go

```diff
@@ -58,9 +58,9 @@ func (w *LocalClient) Write(ctx context.Context, name string, reader io.Reader,
 	if err != nil {
 		switch {
 		case errors.Is(err, fs.ErrNotExist):
-			return NoSuchDirectoryError{path: absPath}
+			return noSuchDirectoryError{path: absPath}
```

Comment:
why replace fs.ErrNotExists with noSuchDirectoryError which pretends to be fs.ErrNotExist ?

People checking Is() will not see the difference.